mailpoet/assets/js/src/newsletter_editor/blocks/coupon.ts

```diff
@@ -268,6 +272,12 @@ Module.CouponBlockSettingsView = base.BlockSettingsView.extend({
       // set selected code from available
       this.model.set(
         'code',
+        this.$('.mailpoet_field_coupon_existing_coupon')
+          .find(':selected')
+          .html(),
```

Comment:
Can you use `.text()` here?
instead of `.html()` 

Difference between them [here](https://www.educative.io/answers/what-is-the-difference-between-html-and-text-in-jquery)

Reply:
👍 I will change it.